firmware/projects/FrontController/inc/simulink.h

```diff
@@ -0,0 +1,29 @@
+/// @author Blake Freer
+/// @date 2024-03-14
+
+#include "../vehicle_control_system/src/vehicle_control_system-autogen/controller_autogen_ert_rtw/controller_autogen.h"
```

Comment:
Is the relative parent directory in the file path needed?

Reply:
I should check, you're right that the CMake include directories should handle this

Reply:
see latest commit, this path has been shortened